libdevcore/Whiskers.cpp

```diff
@@ -99,15 +99,43 @@ void Whiskers::checkParameterUnknown(string const& _parameter) const
 	);
 }
 
+namespace {
```

Comment:
Please fix formatting:
 - `{` on a line of its own
 - no indentation inside namespaces
 - `const` at the right side of the type
 - `)` on a line of its own
 - single indentation depth for function parameters